manifest.json

```diff
@@ -3,9 +3,9 @@
     "id": "zap",
     "packaging_format": 1,
     "description": {
-        "en": "Zap is a full featured social network application with identity/content mirroring across multiple servers (nomadic identity) but no federate with non-nomadic servers."
+        "en": "Zap is a full featured social network application with identity/content mirroring across multiple servers (nomadic identity) federating over ActivityPub and Zot6 protocols."
     },
-    "version": "2.6~ynh1",
+    "version": "20.04.27~ynh1",
```

Comment:
Can you confirm this is the real version number used on the upstream ?

Reply:
@alexAubin yes, I just confirmed it with the developer.